### ISSUE TITLE
Unable to save product through product field's HUD

### ISSUE THREAD

**Issue by engram-design** [craftcms/commerce#418]
### Description
Double-clicking a HUD from a product field allows you to edit the product. I'm getting an error preventing saving:

```
"Call to a member function getTimestamp() on boolean"
```

Full details:
```
Request URL: http://commerce.local/index.php?p=office/actions/elements/save-element
Request Method: POST
Status Code: 500 Internal Server Error
siteId: 1
elementId: 378
namespace: editor_i73INtpndj
elementId: 378
siteId: 1
editor_i73INtpndj[title]: TEST
editor_i73INtpndj[slug]: test
editor_i73INtpndj[postDate][date]: 13/7/2018
editor_i73INtpndj[postDate][timezone]: Australia/Melbourne
editor_i73INtpndj[postDate][time]: 11:23 am
editor_i73INtpndj[postDate][timezone]: Australia/Melbourne
editor_i73INtpndj[expiryDate][date]: 
editor_i73INtpndj[expiryDate][timezone]: Australia/Melbourne
editor_i73INtpndj[expiryDate][time]: 
editor_i73INtpndj[expiryDate][timezone]: Australia/Melbourne
editor_i73INtpndj[taxCategoryId]: 1
editor_i73INtpndj[shippingCategoryId]: 1
editor_i73INtpndj[availableForPurchase]: 
editor_i73INtpndj[availableForPurchase]: 1
editor_i73INtpndj[freeShipping]: 
editor_i73INtpndj[promotable]: 
editor_i73INtpndj[promotable]: 1
editor_i73INtpndj[fieldLayoutId]: 57
editor_i73INtpndj[variants][379][sku]: TEST
editor_i73INtpndj[variants][379][price]: 0
editor_i73INtpndj[variants][379][hasUnlimitedStock]: 
editor_i73INtpndj[variants][379][hasUnlimitedStock]: 1
editor_i73INtpndj[variants][379][minQty]: 
editor_i73INtpndj[variants][379][maxQty]: 
```

### Additional info

- Commerce version: dev-develop

**Comment by nfourtythree** [comment]
Hey @engram-design 

I have just pushed up a fix for this, which should be included in the next release.

**Comment by lukeholder** [comment]
Fixed in 6d03be53d885395400a80e4caa298c378c835825
